cypress/integration/User Management/BL_00_ReceptionistEdit.js

```diff
@@ -0,0 +1,128 @@
+                            },
+                        },
+                    },
+                    "test": (variables) => {
+                        Object.entries(variables).forEach(([key, value]) => {
+                            if (!["phoneNumber", "id", "user", "password"].includes(key)) {
+                                expect(receptionist[key] || receptionist?.user[key]).equals(value);
+                            }
+                        });
+                    },
+                },
+                "GetAdmin": {
+                    "response": {
+                        "data": {
+                            "admin": {
+                                ...receptionist,
+                                "password": null,
+                            },
+                        },
+                    },
+                    "test": ({ userID }) => {
+                        expect(userID).equals(receptionist.user.id.toString(), "Check ID passed");
+                    },
+                },
+            });
+            cy.visitAuthenticated(`/form/admin/${receptionist.user.id}`);
+        });
+    });
+
+    it("Loads our mock data properly into the login details form section for receptionist", () => {
+        cy.fixture("users.json").then(({ receptionist }) => {
+            cy.get("[data-cy=login-firstName-input]")
+                .should("have.value", receptionist.user.firstName);
+            cy.get("[data-cy=login-lastName-input]")
+                .should("have.value", receptionist.user.lastName);
+            cy.get("[data-cy=login-email-input]")
+                .should("have.value", receptionist.user.email);
+            cy.get("[data-cy=login-nextButton]")
+                .should("be.disabled");
+        });
+    });
+
+    it("Can enter data from the user properly into the first form, while checking if bad info is entered the submit button should be disabled", () => {
+        cy.fixture("users.json").then(({ receptionist }) => {
+            cy.get("[data-cy=login-firstName-input]")
+                .clear()
+            cy.get("[data-cy=login-firstName-input]")
+                .fastType("0");
+            cy.get("[data-cy=login-nextButton]")
+                .should("be.disabled");
+            cy.get("[data-cy=login-firstName-input]")
+                .clear();
+            cy.get("[data-cy=login-firstName-input]")
+                .fastType(receptionist.user.firstName);
+            cy.get("[data-cy=login-nextButton]")
+                .should("be.disabled");
+            cy.get("[data-cy=login-password-input]")
+                .should("be.empty")
+                .fastType(0)
+                .clear()
+                .fastType("test1234")
+                .should("have.value", "test1234");
+        });
+    });
+
+    it("Properly goes to the next section of the form called User Information and checks for data in stubs to load properly on the second form", () => {
```

Comment:
don't use adjectives to describe tests (e.g. "Properly"). This should be renamed to: "Loads _____ into User Information section of the receptionist editing form" 

You don't need to describe the specific actions that you're performing in a test. The title should reflect what is being tested/the assertions